haiku-apps/nanodot/nanodot-1.1.recipe

```diff
@@ -0,0 +1,46 @@
+SUMMARY="Binary & source for a simple MIDI sequencer"
+DESCRIPTION="\
+              nanodot is a rather minimal pattern-based MIDI\n\
+               sequencer for BeOS R3. Its main purpose in life\n\
+               was/is to teach me (and hopefully others) the\n\
+               basics of writing MIDI apps for BeOS; it makes\n\
+               for a nice drum-programming 'scratchpad', and is a\n\
+               generally entertaining way to waste a few hours."
```

Comment:
The wrapping is not what we like to have. Here it is, corrected, and with a tiny change in the text ("originally written for"):
```
DESCRIPTION="nanodot is a rather minimal pattern-based MIDI sequencer \
originally written for BeOS R3. Its main purpose in life was/is to teach me \
(and hopefully others) the basics of writing MIDI apps for BeOS; it makes for \
a nice drum-programming 'scratchpad', and is a generally entertaining way to \
waste a few hours."
```